doc/source/ray-air/examples/feast_example.ipynb

```diff
@@ -0,0 +1,1512 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Integrate Ray Air with Feast feature store"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# !pip install feast==0.20.1 ray[air]>=1.13 xgboost_ray"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "INyNIaeB1Kza"
+   },
+   "source": [
+    "In this example, we showcase how to use Ray Air with Feast feature store, leveraging both historical features for training a model and online features for inference.\n",
+    "\n",
+    "The task is adapted from [Feast credit scoring tutorial](https://github.com/feast-dev/feast-aws-credit-scoring-tutorial). In this example, we train a xgboost model and run some prediction on an incoming loan request to see if it is approved or rejected. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "sBC9CCrpzQLF"
+   },
+   "source": [
+    "Let's first set up our workspace and prepare the data to work with."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {
+    "id": "DcPIskZlzSal"
+   },
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "WORKING_DIR = os.environ.get(\n",
+    "    \"WORKING_DIR\", os.path.expanduser(\"~/ray-air-feast-example/\")\n",
+    ")\n",
+    "os.environ[\"WORKING_DIR\"] = WORKING_DIR"
```

Comment:
Hmm, why do we have to do this?

Reply:
wrong link?

Reply:
updated https://stackoverflow.com/a/40932507/14961124

Reply:
nice! updated!